tests/loops/test_loops.py

```diff
@@ -912,8 +912,9 @@ def val_dataloader(self):
 
 
 @RunIf(min_torch="1.8.0")
-@pytest.mark.parametrize("persistent_workers", (False, True))
-def test_workers_are_shutdown(tmpdir, persistent_workers):
+@pytest.mark.parametrize("should_fail", [False, True])
+@pytest.mark.parametrize("persistent_workers", [False, True])
+def test_workers_are_shutdown(tmpdir, should_fail, persistent_workers):
```

Comment:
The test takes 50s to run on my machine. Do you have any ideas how we could improve that?

Reply:
Definitely any issue. I don't have a good idea. Re-creating the processes is quite expensive on Mac.

Reply:
Is it safe to assume that the slowness only occurs if `persistent_workers=False`? Or are you seeing a problem with the value is `True`?

If the problem only occurs on `persistent_workers=False` then I wonder why we haven't run across this problem before as this is the default value of `persistent_workers`.

Reply:
it has num_workers=1 so yes, no surprise :) 
If we really want to test this, we have to either take the 50s hit or use unittest mock to check the calls as we expect them (and don't explicitly spawn processes).

Reply:
Done !